src/button/index.js

```diff
@@ -92,10 +92,10 @@ function _Button (props) {
     <Component
       className={classNames(
         classes.button,
-        primary ? classes.primary : null,
-        secondary ? classes.secondary : null,
-        secondaryLight ? classes.secondaryLight : null,
-        disabled ? classes.disabled : null,
+        primary ? classes.buttonPrimary : null,
+        secondary ? classes.buttonSecondary : null,
+        secondaryLight ? classes.buttonSecondaryLight : null,
+        disabled ? classes.buttonDisabled : null,
```

Comment:
I'd recommend removing the `button` prefix because overriding these classes later will look like:
```js
    <Button
      classes={{
        buttonPrimary: classes.primary, 
        buttonDisabled: classes.disabled
      }}
    >
```

Reply:
I have reverted these changes, thanks for clarifying!